docs/sphinx/reference-frontend-api.rst

```diff
@@ -595,3 +599,27 @@ Functions
 .. function:: void obs_frontend_reset_video(void)
 
    Reloads the UI canvas and resets libobs video with latest data from profile.
+
+---------------------------------------
+
+.. function:: void obs_frontend_set_tbar_position(int position)
+
+   Sets the value of the transition bar.
+
+   :param position: The position to set the t-bar to
+   :value Range: * 0 - 1023
```

Comment:
I think this should be:
```
:param position: The position to set the T-bar to, with a value in 0-1023.
:type position: int
```

The `:value` syntax appears to be meant for ":value: initial value of the variable (text)" rather than possible values.  The `:value` field type seems to be meant for Python rather than C/C++.

https://www.sphinx-doc.org/en/3.x/usage/restructuredtext/domains.html#info-field-lists